Source/Urho3D/Audio/SoundSource.cpp

```diff
@@ -358,14 +361,23 @@ void SoundSource::Update(float timeStep)
 
 void SoundSource::Mix(int dest[], unsigned samples, int mixRate, SpeakerMode mode, bool interpolation)
 {
-    if (!position_ || (!sound_ && !soundStream_) || (!IsEnabledEffective() && node_ != nullptr))
+    if (!position_ || (!sound_ && !soundStream_))
         return;
 
+     if (node_ != nullptr)
+     {
+         if (!IsEnabledEffective())
```

Comment:
This bit is strange. So when `node_` is null code will proceed to mix the sound. Is this a valid usecase?